distributions/Dist_mkb452.py

```diff
@@ -3,22 +3,22 @@
 
 class Dist_mkb452(BaseDistribution):
 	def __init__(self):
-		self.f_max = 1
-		self.x_min = -np.inf
-		self.x_max = np.inf
+		self.f_max = 0.5
+		self.x_min = 2
+		self.x_max = 5
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		y = (-np.square(x))/2
-		return ((1/(np.sqrt(2*np.pi)))np.exp(y))
+		y = (1/0.000022737)
+		return (y*np.exp(-x,3)*np.sin(x))
```

Comment:
syntax error with exp(-x,3)

Reply:
I completely forgot about the np.power, hopefully the code is correct now. When I try mydist.rvs(1000) now though, I get a different error. Instead of syntax, it says, "TypeError: return arrays must be of ArrayType". I'm not sure how to proceed, I tried different solutions and I get the same error.